src/routes/popup/notifications.tsx

```diff
@@ -83,24 +89,66 @@ export default function Notifications() {
   const formatTxMessage = async (notifications) => {
     const formattedTxMsgs = [];
     for (const notification of notifications) {
-      const ticker =
-        notification.isAo && notification.transactionType !== "Message"
-          ? await getTicker(notification.tokenId)
-          : notification.tokenId;
-      let formattedMessage: string;
-      if (notification.transactionType === "Sent") {
-        formattedMessage = `Sent ${Number(notification.quantity).toFixed(
-          2
-        )} ${ticker} to ${
-          notification.ao
-            ? findRecipient(notification)
-            : formatAddress(notification.node.recipient, 4)
-        }`;
-      } else if (notification.transactionType === "Received") {
-        formattedMessage = `Received ${Number(notification.quantity).toFixed(
-          2
-        )} ${ticker} from ${formatAddress(notification.node.owner.address, 4)}`;
-      } else if (notification.transactionType === "Message") {
+      let formattedMessage;
+      if (notification.transactionType !== "Message") {
+        let ticker;
+        let quantityTransfered;
+        if (notification.isAo) {
+          // handle ao messages/sents/receives
+          let token = await fetchTokenByProcessId(notification.tokenId);
+          if (!token) {
+            ticker = formatAddress(notification.tokenId, 4);
+            quantityTransfered = notification.quantity;
+          } else {
+            ticker = token.Ticker;
+            quantityTransfered = balanceToFractioned(
+              Number(notification.quantity),
+              {
+                id: notification.tokenId,
+                decimals: token.Denomination,
+                divisibility: token.Denomination
+              }
+            );
+          }
+        } else if (notification.transactionType !== "Transaction") {
+          let token = await fetchTokenById(notification.tokenId);
+          if (!token) {
+            ticker = formatAddress(notification.tokenId, 5);
+            quantityTransfered = extractQuantityTransferred(
+              notification.node.tags
+            );
+          } else if (token.ticker !== "AR") {
```

Comment:
Same problem here if the user decides to remove all the warp tokens from settings.